### ISSUE TITLE
applications presentation at the user portal page

### ISSUE THREAD

**Issue by panosnethood** [mazi-project/portal#18]
There are many cases in which the application title does not align correctly inside the corresponding box.

I understand that this is a little complicated, but perhaps a nice solution is to show the applications in rows, as in the smartphone view ... it is easier to browse like this.

**Comment by panosnethood** [comment]
It would be also nice if one could choose in addition to the type also the colous of the "application box". Otherwise the portal looks too similar even if the overall framing is different.

**Comment by andreasunteidig** [comment]
We did not have time to discuss this in London, but I would like to to do/bootstrap some focused work on the interface design. This is a mayor part of the toolkit and I think there's a lot more potential from a design perspective. Especially if we start thinking in terms of themes, etc. maybe a topic for post-review Volos?

**Comment by panosnethood** [comment]
Yes, and this discussion is also related to this issue, #37, which could be a complementary option to a set of "native" MAZI themes (in addition to the whole variety of wordpress themes, which might have some limitations in integrating all available functionality but will be also "heavier")
